src/data/dataloader.cpp

```diff
@@ -458,6 +458,23 @@ Labelset loadFileWithCache(std::filesystem::path filePath) {
     );
 }
 
+Labelset loadFromDataset(const Dataset& dataset) {
+    Labelset res;
+    res.entries.reserve(dataset.entries.size());
+
+    int count = 0;
+    for (const Dataset::Entry& entry : dataset.entries) {
+        Labelset::Entry label;
+        label.position = entry.position;
+        label.text = entry.comment.value_or("MISSING LABEL");
+        // @TODO: make is possible to configure this identifier?
```

Comment:
@Roxeena let's discuss this or maybe handle it in the label rewrite?

Reply:
I think it is fine for now, but we can discuss it in the label rewrite work.